src/main/webapp/WEB-INF/web.xml

```diff
@@ -97,12 +97,12 @@
 		<url-pattern>/*</url-pattern>
 	</filter-mapping>
 
-    <filter-mapping>
-        <filter-name>CrossSiteScriptingPreventionFilter</filter-name>
-        <url-pattern>/*</url-pattern>
-    </filter-mapping>
+    <!-- <filter-mapping>
+		 <filter-name>CrossSiteScriptingPreventionFilter</filter-name>
+		 <url-pattern>/*</url-pattern>
+	 </filter-mapping> -->
```

Comment:
This has been fixed, remove this

Reply:
I removed this filter because multimedia was being corrupted.

Is this the fix you're referring to?

Reply:
Done.